src/styles/main.scss

```diff
@@ -1 +1,63 @@
-// put code here
+@import './utils/variables';
+
+.clock {
+  display: flex;
+  height: $clock-size;
+  width: $clock-size;
+  border: 6px dotted black;
+  border-radius: 50%;
+  margin-inline: auto;
+  margin-block: 10vmin;
+  position: relative;
+
+  &__second-hand {
+    display: flex;
+    width: 10px;
+    height: 50%;
+    background-color: green;
+    position: absolute;
+    transform: translateX(-50%);
+    top: 0;
+    left: 50%;
+    transform-origin: center bottom;
+    animation: second-hand 60s forwards 1 linear;
+  }
+
+  &__minute-hand {
+    display: flex;
+    width: 20px;
+    height: 25%;
+    background-color: blue;
+    position: absolute;
+    transform: translateX(-50%);
+    top: 25%;
+    left: 50%;
+    transform-origin: center bottom;
+    animation: minute-hand 5s forwards 1 linear;
+  }
```

Comment:
your hands have a lot of repeated styles. move them either to a `placeholder` or to a common class selector